pkg/slayers/scion.go

```diff
@@ -591,16 +611,33 @@ func (s *SCION) SetSrcAddr(src net.Addr) error {
 	return err
 }
 
-// @ trusted
-// @ requires false
-func parseAddr(addrType AddrType, raw []byte) (net.Addr, error) {
+// @ requires  sl.AbsSlice_Bytes(raw, 0, len(raw))
```

Comment:
are you sure we are always able to provide this permission `sl.AbsSlice_Bytes(raw, 0, len(raw))` on every call of this function (and on every call of `SrcAddr` and `DstAddr`)?